tools/bridge/bridge/confirmation_task_planner.py

```diff
@@ -67,6 +64,11 @@ def process_home_bridge_events(self) -> None:
                 logger.debug("Received home bridge event")
                 self.recorder.apply_proper_event(event)
 
+    def process_control_events(self) -> None:
+        while True:
+            event = self.control_queue.get()
+            self.recorder.apply_control_event(event)
+
```

Comment:
As I said, I think all of the process event functions can be unified. But for now that's also ok.